provisioning/security/tpm-provider-emulator/src/main/java/com/microsoft/azure/sdk/iot/provisioning/security/hsm/SecurityProviderTPMEmulator.java

```diff
@@ -102,6 +102,14 @@ public SecurityProviderTPMEmulator(String registrationId) throws SecurityProvide
         srkPublic = createPersistentPrimary(tpm, SRK_PERSISTENT_HANDLE, TPM_RH.OWNER, SRK_TEMPLATE, "SRK");
     }
 
+    public static SecurityProviderTPMEmulator createProviderFromKey(String authenticationKey) throws SecurityProviderException
```

Comment:
Leave a TODO somewhere in here so you don't forget to add this same constructor to the SecurityProviderTpmHsm class as well